docs/intro.asciidoc

```diff
@@ -7,6 +7,11 @@ It has built-in support for popular platforms and frameworks, and an API for cus
 The Agent also supports <<distributed-tracing-guide,distributed tracing>> for all outgoing requests.
 This enables you to analyze performance throughout your microservice architecture -- all in one view.
 
+[NOTE]
+====
+The Elastic APM RUM JavaScript Agent is not compatible with {serverless-docs}[{serverless-full}].
```

Comment:
Does this sound ok?

```suggestion
The Elastic APM RUM JavaScript Agent is _not_ compatible with {serverless-docs}[{serverless-full}] -- it cannot send data to the APM endpoint for serverless projects.
```

<img width="875" alt="image" src="https://github.com/user-attachments/assets/a86c0ea6-ac9a-4886-a3c1-4f8719c9d53a" />

Reply:
Looks good to me.